lib/nimble_template/addons/github.ex

```diff
@@ -89,16 +89,22 @@ defmodule NimbleTemplate.Addons.Github do
   end
 
   @impl true
-  def do_apply(
-        %Project{
-          web_project?: web_project?,
-          mix_project?: mix_project?,
-          erlang_version: erlang_version,
-          elixir_version: elixir_version
-        } = project,
-        opts
-      )
-      when is_map_key(opts, :github_wiki) do
+  def do_apply(%Project{} = project, opts) when is_map_key(opts, :github_wiki) do
```

Comment:
Just wondering where does it fallback to when `github_wiki` does not present in the options? 💭

Reply:
Hi Ros, if we don't pass that key then it will look for another key

![image](https://user-images.githubusercontent.com/11751745/148039911-6453c3c3-7ee3-402a-8fdd-41a76af4adc0.png)

In case no key is passed into the `Github apply` then the exception will be raised.

In that case, there will be something wrong internally and we need to fix that :)

Reply:
I see. Seems the opts is a map, so we also could do something like this


```suggestion
  def do_apply(%Project{} = project, %{github_wiki: true}) do
```
🤔 💡

Reply:
Hi @rosle I don't remember much the reason the team used `when is_map_key(opts, :github_wiki)` let me check the git log again :D

Reply:
I have opened a chore PR for that FYI @rosle https://github.com/nimblehq/elixir-templates/pull/152